salvo/src/lib/benchmark/fully_dockerized_benchmark.py

```diff
@@ -7,7 +7,7 @@
 
 import logging
 
-from src.lib.benchmark.base_benchmark import BaseBenchmark
+from src.lib.benchmark.base_benchmark import (BaseBenchmark, get_docker_volumes)
```

Comment:
Sorry I must have missed this before, can we change the import to one that imports the module rather than classes and methods?

```python
from src.lib.benchmark import base_benchmark

# And then in the code:
base_benchmark.BaseBenchmark()
base_benchmark.get_docker_volumes()
```

The style guide recommends we always import modules to make it easier for readers to identify the source of each symbol.

https://github.com/google/styleguide/blob/gh-pages/pyguide.md#22-imports

Reply:
This comment doesn't seem to be addressed yet. In addition to this location, we should address the same in `salvo/src/lib/test_docker_image.py` also which was changed in one of the recent commits.

Reply:
Yep.  I can change this.